testbed/README.md

```diff
@@ -39,3 +45,93 @@ interfaces the `NewTestCase` function takes as parameters.
 * `TestResultsSummary` - Records itemized test case results plus a summary of one category of testing.
   * `PerformanceResults` - Implementation of `TestResultsSummary` with fields suitable for reporting performance test results.
   * `CorrectnessResults` - Implementation of `TestResultsSummary` with fields suitable for reporting data translation correctness test results.
+
+## Step-by-Step Instruction for development
+
+Generally, when designing a test for new exporter and receiver components, by taking use of existing implementation of testbed, developers should mainly focus on designing and implementing the components with yellow background in the diagram above:
```

Comment:
```suggestion
Generally, when designing a test for new exporter and receiver components, developers should mainly focus on designing and implementing the components with yellow background in the diagram above as the other components are implemented by the testbed framework :
```

Reply:
> Any reason for the Untitled.md?

Sorry, my fault, I will tackle it! Thank you!

Reply:
Thank you for your suggestions @anuraaga , I have already revised the content and made new commits!